src/main/java/study/step4/Cars.java

```diff
@@ -0,0 +1,69 @@
+package study.step4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private List<Car> cars = new ArrayList<>();
+    private List<Car> winners = new ArrayList<>();
+
+    public Cars() {
+
+    }
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    /* 입력값 세팅 */
+    public void split(String inputCarNames) {
+        String[] carNames = inputCarNames.split(",");
+        for (String carName : carNames) {
+            Car car = new Car(carName);
+            cars.add(car);
+        }
```

Comment:
해당 메서드는 외부에 공개될 필요가 없는 것 같아요!
inputCarNames를 파라미터로 받는 주 생성자를 하나 더 생성하여
이름을 받는만큼 자동차를 생성하도록 하면 어떨까요 :) ?

```suggestion
    public Cars(List<Car> cars) {
        this.cars = cars;
    }
    
    public Cars(String inputCarNames) {
        this.cars = split(inputCarNames);
    }

    private void split(String inputCarNames) {
        List<Car> cars = new ArrayList<>();
        String[] carNames = inputCarNames.split(",");
        for (String carName : carNames) {
            cars.add(new Car(carName));
        }
        return cars;
```

Reply:
넵 더 효율이 좋아보이네요. 좋은 의견 감사합니다 👍